Mailbox/Person.cs

```diff
@@ -1,4 +1,66 @@
-﻿namespace Mailbox
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Mailbox
 {
+#pragma warning disable CA1067 // Override Object.Equals(object) when implementing IEquatable<T>
+    public struct Person : IEquatable<Person>
+#pragma warning restore CA1067 // Override Object.Equals(object) when implementing IEquatable<T>
+    {
+        private string? FirstName { get; set; }
+        private string? LastName { get; set; }
```

Comment:
Given the null checks in the constructor these could be made readonly non-nullable properties.
```suggestion
        private string LastName { get; }
```